### PR TITLE
Add tool extended drought

### PR DESCRIPTION
**Pull request by mukhtyar**

### CODE REVIEW

File: content/glossary/baseflow.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Baseflow
+---
+
+Portion of the stream flow that is not from precipitation and results from seepage of water from the ground.
+
+Baseflow projections on Cal-Adapt are produced using a land surface and hydrology model known as the [Variable Infiltration Capacity](/help/glossary/) (VIC) model.
```
Comment (mukhtyar) [inline content/glossary/baseflow.md]: add hash to glossary link

File: content/glossary/drought-scenario.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: Drought Scenarios
+---
+
+Two drought scenarios were constructed for California’s Fourth Climate Change Assessment to investigate implications of an extended dry spell:
+
+- Late 21st Century Drought: This scenario represents a late century dry spell from 2051–2070 identified from 
+    the HadGEM2-ES RCP 8.5 simulation. The extended drought scenario is based on the average 
+    annual precipitation over 20 years. This average value equates to 78% of historical median 
+    annual precipitation averaged over the North Coast and Sierra California Climate Tracker regions..
```
Comment (mukhtyar) [inline content/glossary/drought-scenario.md]: remove extra period

File: content/glossary/evapotranspiration.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: Evapotranspiration
+---
+
+Water that is evaporated from the surfaces and transpired from plants.
+
+Evapotranspiration projections on Cal-Adapt are produced using a land surface and hydrology model known as the [Variable Infiltration Capacity](/help/glossary/) (VIC) model.
```
Comment (mukhtyar) [inline content/glossary/evapotranspiration.md]: add glossary link to vic

File: content/glossary/runoff.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+title: Runoff
+---
+
+Water that is discharged into the streams and largely results from precipitation and melting of snow.
+
+Runoff projections on Cal-Adapt are produced using a land surface and hydrology model known as the [Variable Infiltration Capacity](/help/glossary/) (VIC) model.
```
Comment (mukhtyar) [inline content/glossary/runoff.md]: add glossary link to vic

File: content/tools/data.js [hunk 5]
```diff
@@ -216,13 +216,39 @@ const tools = [
   },
   {
     title: 'Extended Drought',
```
Comment (mukhtyar) [inline content/tools/data.js]: Update title to "Extended Drought Scenarios"

File: content/tools/data.js [hunk 6]
```diff
@@ -216,13 +216,39 @@ const tools = [
...
-    desc: `Projections of temperature, precipitation and a set of VIC variables from two 30 year 
-    drought periods.`,
+    desc: `Explore projections of temperature, precipitation and a set of hydrological variables 
+      from two 20 year drought periods.`,
     categories: ['Temperature', 'Precipitation'],
```
Comment (mukhtyar) [inline content/tools/data.js]: add category `Snowpack`

File: src/routes/tools/extended-drought/index.svelte [hunk 7]
```diff
@@ -0,0 +1,287 @@
...
+    console.groupCollapsed("STORE UPDATES");
+    console.table($dataStore);
+    console.table($locationStore);
+    console.groupEnd();
+  }
+
+  async function update() {
+    if (!appReady) return;
+    try {
+      dataStore.set(null);
```
Comment (mukhtyar) [inline src/routes/tools/extended-drought/index.svelte:137]: remove this once the stats panel bug is fixed

File: content/tools/data.js (hunk 6 above)
Comment (mukhtyar) [reply content/tools/data.js]: fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/125/commits/4a775d5f11f50beae431b50793e45c76fb1d40d5

File: content/glossary/drought-scenario.md (hunk 2 above)
Comment (mukhtyar) [reply content/glossary/drought-scenario.md]: fixed in  https://github.com/berkeley-gif/caladapt-website-2021/pull/125/commits/6e747fb64773a30dd1999a9464f1edd8ab09ccf9

File: content/glossary/baseflow.md (hunk 1 above)
Comment (clhenrick) [reply content/glossary/baseflow.md]: @mukhtyar to link to any term in the glossary use a hash version the glossary term's `metadata.title` and replace all spaces with "-" and convert it to lowercase. Then you can use the glossary path and hash term, e.g.

```
/help/glossary/#variable-infiltration-capacity-(vic)
```

File: src/routes/tools/extended-drought/_ChartTitle.svelte [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+<Button
+  class="btn-change-location"
+  size="small"
+  icon="{Location16}"
+  kind="ghost"
+  on:click="{loadLocation}"
+>
+  Change Location
+</Button>
+<div class="h3">
```
Comment (clhenrick) [inline src/routes/tools/extended-drought/_ChartTitle.svelte]: I'm thinking that semantically we might want to use `<h2>` for the chart title, given that it is sort of the title for the dashboard. They could still be styled using the `h3` class though.

**Review by clhenrick — APPROVED** [review]
Looks good to me, the annotations render nicely. The only adjustment I would make is to bump up the font-size slightly for the left and right arrows that surround "Extended Drought Period (20 years)" as they're a little small.

File: content/glossary/runoff.md (hunk 4 above)
Comment (mukhtyar) [reply content/glossary/runoff.md]: fixed via https://github.com/berkeley-gif/caladapt-website-2021/pull/125/commits/69cb77de4d70c5bd1c8af193d3592b86a0a44ea7

File: content/glossary/baseflow.md (hunk 1 above)
Comment (mukhtyar) [reply content/glossary/baseflow.md]: fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/125/commits/69cb77de4d70c5bd1c8af193d3592b86a0a44ea7

**Comment by mukhtyar** [comment]
Closing this PR. It was merged into the new `develop` branch using the foll:
```
git fetch origin
git checkout develop
git merge --no-ff add-tool-extended-drought
git push origin develop
```

**Pull request closed without merging**